<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="colormap_combo_box">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>RdYlBu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>YlOrRd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>inferno</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>plasma</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>magma</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>coolwarm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>autumn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>spectral</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rainbow</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Colormap</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="num_stars_spin_box">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Stars</string>
    </property>
   </widget>
   <widget class="QPushButton" name="animate_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Animate</string>
    </property>
   </widget>
   <widget class="QWidget" name="mpl_window" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="mplvl">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QCheckBox" name="tan_vel_check_box">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tangential Velocities</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="star_trails_check_box">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Star Trails</string>
    </property>
   </widget>
   <widget class="QPushButton" name="terminate_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="track_ejecta_check_box">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Ejecta</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
